orchestrator.py

```diff
@@ -79,6 +137,12 @@ def on_message(self, client, userdata, msg):
             )
         elif topics.Camera.set_overlay.matches(msg.topic):
             config.set_overlay(json.loads(str(msg.payload.decode("utf-8"))))
+        elif topics.WirelessModule.all().start.matches(msg.topic):
+            self.currently_logging = True
+        elif topics.WirelessModule.all().data.matches(msg.topic):
+            self.currently_logging = True
```

Comment:
Is this for backup? Since  we should only be publishing on the `data` topic if the `start` topic has been triggered, in which case `self.currently_logging` is already set to True

Reply:
It's so that if the cameras are turned on while data is already being published from the WM it's state isn't out of sync.